### PR TITLE
add import coords protocol

### LINKED ISSUE #301: Import coordinates from particles.star
**Is your feature request related to a problem? Please describe.**
In several occasions, users wanted to import a particles.star file (coming from refinements) done outside scipion with relion or CS. But they are interested only in getting the coordinates.

Generic Import coordinates does not fit in this approach since it expects having  1 metadata file per coordinate file.

**Describe the solution you'd like**
A dedicated "relion import coordinates" protocol to deal with particles.star files

**Describe alternatives you've considered**
Another option might be to use import particles, maybe skipping missing binaries, which for this case are not needed?

Thoughts?

### PR DESCRIPTION
**Pull request by azazellochg**
Closes #301

### CODE REVIEW

File: relion/convert/convert31.py
Comment (delarosatrevin) [inline relion/convert/convert31.py:621]: Maybe it is more clear in the following way:

```python
hasMicId = hasattr(firstRow, 'rlnMicrographId')
```

File: relion/convert/convert31.py
Comment (delarosatrevin) [inline relion/convert/convert31.py:625]: Why the first coordinate is treated separated from the rest in the loop (also duplicating some code)

**Review by delarosatrevin — COMMENTED** [review]
Is there any test for this protocol?

File: relion/convert/convert31.py
Comment (azazellochg) [reply relion/convert/convert31.py:625]: Because of self.createExtraLabels(coord, firstRow, extraLabels) that needs to create extra objects first

**Comment by azazellochg** [comment]
Test added!

**Comment by azazellochg** [comment]
I'll merge this tonight if no more comments

**Comment by delarosatrevin** [comment]
> I'll merge this tonight if no more comments

I will test a bit later and let you know if I find something. Just curious...what is the exact time deadline before merging? :)

**Comment by azazellochg** [comment]
Not another 2 weeks? :)

**Comment by delarosatrevin** [comment]
13 days are not yet two weeks :P 
Jokes apart, I will check this to merge the PR....I'm sorry for the delay.

**Review by delarosatrevin — APPROVED** [review]
Nice addition!

**Comment by azazellochg** [comment]
Thanks! I thought it was easy to do and requested long ago

**Comment by delarosatrevin** [comment]
You are right, I'm sorry for the delay.

**Pull request merged**
